src/50_Samples_%26_Templates/Hotel.html

```diff
@@ -0,0 +1,274 @@
+  }
+  #nearby-attractions li.fancy:before {
+    content: "🎩";
+  }
+  #nearby-attractions li.theatre:before {
+    content: "🎭";
+  }
+</style>
+</head>
+<body>
+<!-- #### Hero images -->
+<!--
+  The `amp-carousel` element is used to add hero images for the property. Learn more about configuring the element [here](/components/amp-carousel).
+-->
+<div>
+    <h5>Buckingham Palace Guest Rooms</h5>
+    <div class="star-rating"><span>★★★★☆</span> 4.3 stars from 250 reviews</div>
+    <amp-carousel id="hero-images" width="400" height="300" layout="responsive" type="slides">
+        <div>
+            <amp-img src="/img/001-2benny.jpg" layout="fill"></amp-img>
+            <div class="caption">Photo by <a href="https://flic.kr/p/rAGfoE">2benny</a> / <a href="https://creativecommons.org/licenses/by/2.0/">CC BY 2.0</a></div>
+        </div>
+        <div>
+            <amp-img src="/img/002-joepitha.jpg" layout="fill"></amp-img>
+            <div class="caption">Photo by <a href="https://flic.kr/p/ew3kma">Joe Pitha</a> / <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC BY-SA 2.0</a></div>
+        </div>
+        <div>
+            <amp-img src="/img/003-muffinn.jpg" layout="fill"></amp-img>
+            <div class="caption">Photo by <a href="https://flic.kr/p/cuDKc3">muffinn</a> / <a href="https://creativecommons.org/licenses/by/2.0/">CC BY 2.0</a></div>
+        </div>
+        <div>
+            <amp-img src="/img/004-robinhawkes.jpg" layout="fill"></amp-img>
+            <div class="caption">Photo by <a href="https://flic.kr/p/bw8B7r">Robin Hawkes</a> / <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC BY-SA 2.0</a></div>
+        </div>
+    </amp-carousel>
+    <p>Located next to London's Belgrave district Buckingham Palace offers a unique opportunity to stay in a piece of England's history. Traditionally the home of the UK's ruling sovereign since 1837 a small number of rooms are now available for rent. Spacious
+        high-ceilinged rooms offer four poster beds and views across St James Park. WiFi is limited.</p>
+</div>
+<!-- #### User status -->
+<!--
+  The `amp-access` element checks the user's current session. The JSON response includes the user's loyalty status and how many free nights they have earned.
+  ```json
+  {
+    "username":"test-user",
+    "status":"Gold",
+    "freenights":2
+  }
+  ```
+-->
+<div>
+    <section amp-access="status = 'Gold'">
+        <template amp-access-template type="amp-mustache">
+            <div class="loyaltymessage">Congratulations <strong>{{username}}</strong>, your PAMPered <strong>{{status}}</strong> status has earned you <strong>{{freenights}}</strong> free nights you can apply to this stay.</div>
+        </template>
+    </section>
+</div>
+<!-- #### Dates of stay -->
+<!--
+  An amp-form is used to allow the user to change the dates of their stay. Learn more about handling the result of a form submission in the [form example](/components/amp-form).
```

Comment:
Can you add a note that native date pickers work in AMP as well (just to remind people that they don't need a fancy calendar picker).